addons/website_forum/models/forum.py

```diff
@@ -500,6 +500,15 @@ def write(self, vals):
         return res
 
     @api.multi
+    def post_comment(self):
+        base_url = self.env['ir.config_parameter'].get_param('web.base.url')
+        for post in self:
+            p = post.parent_id or post
+            body = _('<p>A new comment for <i>%s</i> has been posted. <a href="%s/forum/%s/question/%s">Click here to access the post.</a></p>') % \
```

Comment:
in my commit, I put "A new answer" on purpose to benefit from the existing translations. If we introduce a new string, need to update the .pot file